Meadowlark.net/Meadowlark.Net.Core/Middleware/ParsePathMiddleware.cs

```diff
@@ -0,0 +1,77 @@
+using System.Text.RegularExpressions;
+using Meadowlark.Net.Core.Model;
+
+namespace Meadowlark.Net.Core.Middleware;
+
+public static class ParsePathMiddleware
+{
+  private static string Decapitalize(string str)
+  {
+    if (str.Length == 0) return str;
+    if (str.Length == 1) return str.ToLower();
+    return str[0..1].ToLower() + str[1..];
```

Comment:
Although not much of a difference, should the left side be str[0].ToLower() for clarity?

Reply:
Sadly, C# wants str[0] to be a Char type.